pkg/ip/cidr.go

```diff
@@ -30,3 +31,38 @@ func ParseCIDRs(cidrs []string) (valid []*net.IPNet, invalid []string) {
 	}
 	return valid, invalid
 }
+
+// PrefixToIPNet is a convenience helper for migrating from the older 'net'
+// standard library types to the newer 'netip' types. Use this to plug the
+// new types in newer code into older types in older code during the migration.
+func PrefixToIPNet(prefix netip.Prefix) *net.IPNet {
+	return &net.IPNet{
+		IP:   prefix.Addr().AsSlice(),
+		Mask: net.CIDRMask(prefix.Bits(), prefix.Addr().BitLen()),
+	}
```

Comment:
Depending on how `prefix` was constructed, `prefix.Addr()` might have lower bits set, so I'd probably use the masked address here so it matches how `net.IPNet` is used so far, i.e.

```suggestion
	addr := prefix.Masked().Addr()
	return &net.IPNet{
		IP:   addr.AsSlice(),
		Mask: net.CIDRMask(prefix.Bits(), addr.BitLen()),
	}
```

Also we might want to check `prefix.IsValid` and return a nil `*net.IPNet` in case it's not a valid prefix.